src/utils/config.c

```diff
@@ -357,3 +362,113 @@ bool conf_is_list(config_setting_t *conf, const char *file, size_t line,
     }
     return true;
 }
+
+/*
+ * config_group_apply
+ *      Apply a function to a group. Optional `arg` can be used to pass user
+ *      data into the specified function.
+ */
+void config_group_apply(const config_setting_t *options, group_func func, void *arg)
```

Comment:
Could probably use va_args if that's ever necessary.  Don't need to implement that now.